hack/run-kiali-config-template.yaml

```diff
@@ -19,7 +19,7 @@ auth:
   strategy: "anonymous"
 
 deployment:
-  accessible_namespaces: ["**"]
+  accessible_namespaces: ["bookinfo"]
```

Comment:
this assumes when you run run-kiali.sh that you have bookinfo installed - and that is definitely not always the case. You could leave this as an empty list (or explicitly set it to a one-item `istio-system`).

Reply:
Whoops, I edited this in my local testing but forgot to put it back. Thanks for catching it.